daft/functions/ai/__init__.py

```diff
@@ -61,6 +61,8 @@ def embed_text(
     *,
     provider: str | Provider | None = None,
     model: str | None = None,
+    max_retries: int = 3,
+    on_error: str = "raise",
```

Comment:
Not strictly opposed to this, but I'm wondering if we should expose `max_retries` as a configurable option on built-in functions instead of in each provider.